selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/MachinesAsynchronousStartTest.java

```diff
@@ -14,31 +14,46 @@
 import static java.lang.String.format;
 import static java.util.Arrays.asList;
 import static org.eclipse.che.selenium.core.TestGroup.OPENSHIFT;
-import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces.Status.RUNNING;
-import static org.testng.Assert.fail;
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetails.ActionButton.SAVE_BUTTON;
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetails.WorkspaceDetailsTab.MACHINES;
 
 import com.google.inject.Inject;
 import java.util.List;
+import org.eclipse.che.api.core.model.workspace.Workspace;
+import org.eclipse.che.commons.lang.NameGenerator;
 import org.eclipse.che.selenium.core.client.TestWorkspaceServiceClient;
 import org.eclipse.che.selenium.core.executor.OpenShiftCliCommandExecutor;
 import org.eclipse.che.selenium.core.user.DefaultTestUser;
+import org.eclipse.che.selenium.core.webdriver.SeleniumWebDriverHelper;
 import org.eclipse.che.selenium.core.webdriver.WebDriverWaitFactory;
-import org.eclipse.che.selenium.core.workspace.TestWorkspace;
 import org.eclipse.che.selenium.core.workspace.TestWorkspaceProvider;
-import org.eclipse.che.selenium.core.workspace.WorkspaceTemplate;
 import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.eclipse.che.selenium.pageobject.dashboard.NewWorkspace;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.EditMachineForm;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetails;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetailsMachines;
 import org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces;
-import org.openqa.selenium.TimeoutException;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 @Test(groups = OPENSHIFT)
 public class MachinesAsynchronousStartTest {
-  private static final String GET_POD_NAME_COMMAND_COMMAND_TEMPLATE =
-      "get pod --no-headers=true -l che.workspace_id=%s | awk '{print $1}'";
+  private static final String WORKSPACE_NAME = NameGenerator.generate("test-workspace", 4);
+  private static final String MACHINE_NAME = "dev-machine";
+  private static final String IMAGE_NAME = "eclipse/ubuntu_jdk8";
+  private static final String IMAGE_NAME_SUFFIX = NameGenerator.generate("", 4);
+  private static final String NOT_EXISTED_IMAGE_NAME = IMAGE_NAME + IMAGE_NAME_SUFFIX;
+  private static final String SUCCESS_NOTIFICATION_TEST = "Workspace updated.";
   private static final String GET_POD_RELATED_EVENTS_COMMAND_TEMPLATE =
-      "get events --no-headers=true --field-selector involvedObject.name=%s | awk '{print $7 \" \" $8}'";
+      "get event --no-headers=true | grep %s | awk '{print $7 \" \" $8}'";
```

Comment:
Looks like that we getting just workspace related events from the pod events list. If so, next name of conastant would be more appropriate: `GET_WORKSPACE_EVENTS_COMMAND_TEMPLATE`.